spec/queries/draft_and_live_versions_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe Queries::DraftAndLiveVersions do
+  let(:content_id) { SecureRandom.uuid }
+  let(:base_path) { "/vat-rates" }
+
+  let(:payload) {
+    {
+      content_id: content_id,
+      base_path: base_path,
+      update_type: "major",
+      title: "Some Title",
+      publishing_app: "publisher",
+      rendering_app: "frontend",
+      document_type: "guide",
+      schema_name: "guide",
+      locale: "en",
+      routes: [{ path: base_path, type: "exact" }],
+      redirects: [],
+      phase: "beta",
+    }
+  }
+
+  before do
+    3.times do |i|
+      FactoryGirl.create(:content_item,
+        state: 'superseded',
+        content_id: content_id,
+        user_facing_version: i,
+        base_path: base_path
+      )
+    end
+    @item = FactoryGirl.create(:live_content_item, :with_draft,
```

Comment:
Minor - but it'd be a bit more rspec if this was defined via a `let` rather than an instance variable.